src/GraphQL/Execution/SubscriptionExecutionStrategy.cs

```diff
@@ -0,0 +1,310 @@
+using GraphQL.Subscription;
+using GraphQL.Types;
+using GraphQLParser.AST;
+
+namespace GraphQL.Execution
+{
+    /// <summary>
+    /// Executes a subscription.
+    /// </summary>
+    public class SubscriptionExecutionStrategy : ExecutionStrategy
+    {
+        private readonly IExecutionStrategy _baseExecutionStrategy;
+
+        /// <summary>
+        /// Initializes a new instance with a parallel execution strategy for child nodes.
+        /// </summary>
+        public SubscriptionExecutionStrategy()
+            : this(new ParallelExecutionStrategy())
```

Comment:
```suggestion
            : this(ParallelExecutionStrategy.Instance)
```

Saves 1 allocation

Reply:
I thought I did this for a particular reason.  I'm struggling to remember what it was, however.  Hmmm..... seems fine ....

Reply:
Oh.  I did this so that the re-usable fields within `ParallelExecutionStrategy` would not be shared with the subscription execution strategy.  Not a big deal, but seemed to make sense.

Reply:
(Since this is one allocation at load time, and could enhance performance at run-time.)

Reply:
OK. Honestly, 50/50 here, but OK. Let's add explicit comment here about that concern.

Reply:
```suggestion
            : this(new ParallelExecutionStrategy()) // new instance of shared variables within ParallelExecutionStrategy; do not use ParallelExecutionStrategy.Instance
```

Reply:
Merged; please review.